packages/@aws-cdk/aws-stepfunctions/lib/states/pass.ts

```diff
@@ -96,6 +96,16 @@ export interface PassProps {
      * @default No injected result
      */
     readonly result?: Result;
+
+    /**
+     * Parameters pass a collection of key-value pairs, either static values or JSONPath expressions that select from the input.
+     *
+     * @see
+     * https://docs.aws.amazon.com/step-functions/latest/dg/input-output-inputpath-params.html#input-output-parameters
+     *
+     * @default No parameters
+     */
+    readonly parameters?: { [name: string]: any };
```

Comment:
This allows passing of the field. Does it get rendered? Please add a test to prove that it does.

Reply:
I'm also facing an issue with the missing `paramaters` props. Wouldn't it be better to have `PassProps` and `TaskProps` extend `StateProps`?

Reply:
@jogold I agree that `PassProps` and `TaskProps` have a lot of overlap with `StateProps` and could benefit from a refactor, however my goal here is getting `parameters` supported ASAP. As such, I'm only making changes to work within the current codebase design.